.travis.yml

```diff
@@ -28,7 +28,7 @@ script:
 - nix-shell shell-${FIREFOX_VERSION}.nix --command "google-chrome --version"
 - nix-shell shell-${FIREFOX_VERSION}.nix --command "yarn run lint"
 - nix-shell shell-${FIREFOX_VERSION}.nix --command "xvfb-run --server-args=\"-ac -screen 0 1920x1080x16\" yarn run test:once"
-- nix-shell shell-${FIREFOX_VERSION}.nix --command "xvfb-run --server-args=\"-ac -screen 0 1920x1080x16\" yarn run e2e"
+#- nix-shell shell-${FIREFOX_VERSION}.nix --command "xvfb-run --server-args=\"-ac -screen 0 1920x1080x16\" yarn run e2e"
```

Comment:
short comment on why this is outcommented, maybe a link to the underlying Protractor issue?

Reply:
Two related issues are linked now.